.travis.yml

```diff
@@ -30,11 +30,9 @@ matrix:
       - python: 3.5
 
     include:
```

Comment:
Any reason for changing this to trusty? Isn’t that an older image?

Reply:
The database services weren't starting up with bionic. Haven't looked into the differences at all.